pkg/operator/certrotationcontroller/satokensigner_controller.go

```diff
@@ -270,6 +281,29 @@ func (c *SATokenSignerController) syncWorker() error {
 	return nil
 }
 
+//isValidKeyPair checks if a public/private keys match i.e if the tokens signed
```

Comment:
Convention is to include a leading space in comments.

Reply:
Done!